cibyl/sources/source.py

```diff
@@ -74,35 +74,50 @@ def request_handler(*args, **kwargs):
 class Source(AttrDict):
     """Represents a data provider within a system."""
 
-    def __init__(self, name: str = None, driver: str = None, **kwargs):
+    def __init__(self,
+                 name: str = None,
```

Comment:
If the value can be 'None', hint them with 'Optional[str]'.

Reply:
name: Optional[str] = None,
TypeError: 'ABCMeta' object is not subscriptable

Reply:
The package you are looking for is 'typing.', be sure to import that one.

Reply:
Sorry, I imported the wrong one, my fault. Now it's done.